src/main/java/com/manywho/services/sql/services/filter/QueryFilterConditions.java

```diff
@@ -104,16 +102,26 @@ private Object parseParameterObject(TableMetadata tableMetadata, String coulumnN
             case DateTime:
                 return OffsetDateTime.parse(contentValue);
             default:
+                // this special case is for postgres uuid value
```

Comment:
Not sure about this special case being dealt with in the default clause. Could we not add a case to evaluate if contentType is of type string and include it there? Could catch us out otherwise if the sdk were to change. Might be being overly cautious though.

Reply:
This method is transforming the string that comes from the request into a native java object: String, DateTime, Boolean or UUID because the drivers only support those types.
I think I prefer to be less strict here, and let the driver of database throw an exception if a String is not a valid bound for the column, it will probably be an ugly exception but it will tell us exactly what is wrong.